pkg/cli/root.go

```diff
@@ -301,6 +302,15 @@ func validateCacheConfig(cfg *config.Config) error {
 				"remoteCache set to false but cacheDriver config (remote caching) provided for %s,"+
 					"will ignore and use local caching", subPath.RootDirectory)
 		}
+
+		if subPath.GC && subPath.GCDelay <= 0 {
```

Comment:
Or maybe place this is the already existing `validateGC` function.

Reply:
ok, fixed.